MAUI/Polar-Charts/DataLabels.md

```diff
@@ -140,4 +140,124 @@ chart.Series.Add(series);
 
 {% endhighlight %}
 
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## LabelTemplate
+
+The [SfPolarChart]() provides support to customize the appearance of the datalabel by using the [LabelTemplate]() property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfPolarChart>
+        
+        <chart:SfPolarChart.Resources>
+            <DataTemplate x:Key="LabelTemplate1">
+                <StackLayout>
+                    <Frame CornerRadius="50" BorderColor="Black" BackgroundColor="White"
+                           WidthRequest="40" HeightRequest="40" Padding="10">
+                        <Label Text="{Binding Item.Tree}" HorizontalOptions="Center" VerticalOptions="Center" />
+                    </Frame>
+                </StackLayout>
+            </DataTemplate>
+        </chart:SfPolarChart.Resources>
+        . . .
+        <chart:PolarAreaSeries ItemsSource="{Binding PlantDetails}" 
+                               LabelTemplate="{StaticResource LabelTemplate1}"
+                               ShowDataLabels="True"
+                               XBindingPath="Direction"                               
+                               YBindingPath="Tree" />
+    </chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfPolarChart chart = new SfPolarChart();
+    . . .
+    PolarAreaSeries series = new PolarAreaSeries();
+    series.ItemsSource = new ViewModel().Data;
+    series.ShowDataLabels = true;
+    series.XBindingPath="Direction"                               
+    series.YBindingPath="Tree"
+
+    DataTemplate LabelTemplate1 = new DataTemplate(() =>
+    {
+        StackLayout stackLayout = new StackLayout();
+        var frame = new Frame
+        {
+            CornerRadius = 50,
+            BorderColor = Color.Black,
+            BackgroundColor = Color.White,
+            WidthRequest = 40,
+            HeightRequest = 40,
+            Padding = new Thickness(10)
+        };
+
+        var label = new Label
+        {
+            HorizontalOptions = LayoutOptions.Center,
+            VerticalOptions = LayoutOptions.Center
+        };
+        label.SetBinding(Label.TextProperty, "Tree");
+
+        frame.Content = label;
+        stackLayout.Children.Add(frame);
+        return stackLayout;
+    });
+
+    series.LabelTemplate1 = LabelTemplate1;
+    chart.Series.Add(series);
+    this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Label template in MAUI Chart](DataLabel_images/MAUI_polar_datalabel_template.png)
+
+## Formatting Label Context
+
```

Comment:
@EzhilarasanElangovan31 
explain Label context first and then explain label template